extra/io/files/trash/trash-tests.factor

```diff
@@ -0,0 +1,10 @@
+! Copyright (C) 2020 Alexander Ilin.
+! See http://factorcode.org/license.txt for BSD license.
+USING: io.directories io.files.temp io.files.trash kernel tools.test ;
+IN: io.files.trash.tests
+
+{ } [
+    ! temp-file is not used here, because it returns the absolute path, and we
+    ! want to ensure send-to-trash works without giving it the full path.
+    [ "io.files.trash-tests" dup touch-file send-to-trash ] with-temp-directory
```

Comment:
I think you mean ``with-test-directory``, which cleans up after itself?

Reply:
Right! Updated.